c++/Makefile.am

```diff
@@ -576,6 +576,14 @@ capnp_evolution_test_SOURCES = src/capnp/compiler/evolution-test.c++
 
 capnp_afl_testcase_LDADD = libcapnp-test.a libcapnp-rpc.la libcapnp.la libkj.la libkj-async.la
 capnp_afl_testcase_SOURCES = src/capnp/afl-testcase.c++
+
+capnp_llvm_fuzzer_testcase_LDADD = libcapnp-test.a libcapnp-rpc.la libcapnp.la libkj.la libkj-async.la
+capnp_llvm_fuzzer_testcase_SOURCES = src/capnp/fuzz-run.c++
+if HAS_NO_FUZZING_ENGINE
+    capnp_llvm_fuzzer_testcase_SOURCES += src/capnp/fuzz-driver.c++
```

Comment:
Let's remove `fuzz-driver.c++` since we'll test using the actual clang fuzzer in CI.

Reply:
Ok, so no fuzzing driver, no fuzzing target get compiled